### ISSUE TITLE
Make it easy to report a problem in Siskin/Stork/Beagle

### ISSUE THREAD

**Issue by ingoj** [tigase/siskin-im#45]
To improve the situation for XMPP clients on IOS (and other platforms) I think it would be good to have more qualified user feedback. 
For this it would be good to have a bug reporting option within the client that sends all information you need to track down the reported issue, i.e. with a log file. You could add a "Report a problem" item under "More" button and below "Settings" section. This could open a text input field for a short description of the issue the user is facing. Maybe the user can also provide a contact address like mail or JID. 
These reports should be send to one of your servers and not directly uploaded as issue on Github because it may contain sensitive information such as JIDs. Passwords shouldn't be logged, of course. 
Basically sending the problem report can be implemented by sending the report to a special JID using http_upload. This can then generate a mail with a download link for you devs.

**Comment by hantu85** [comment]
Thank you for the suggestion. That is a good idea, however such system would require logging on the device all the time when the app is running and gathering those logs which could contain personal details. That is why it is not so simple thing to do as we will need to prepare logging and gathering them as well as stripping personal details to not invade our users privacy.

**Comment by ingoj** [comment]
For the "log all the time" issue you can make it a config option like "10 mins", "1 hour" or "1 day". 

But yeah, I understand the problems you described. However, I think it would beneficial to have an easy bug reporting. How this can be implemented, is the big question, of course. :-)
